server/src/main.py

```diff
@@ -426,6 +428,15 @@ def housekeeping():
     logger.info(f"Housekeeping completed. Took {housekeeping_end - housekeeping_start:.2f}s")
     return {"message": "Housekeeping completed", "housekeeping_time_s": housekeeping_end - housekeeping_start}
 
+@app.command()
+@fastapi_app.post("/resign")
+def resign_whitelist():
+    """
+    Function to run the cvmfs_server resign command.
+    """
+    logger.info("Running cvmfs_server resign")
+    subprocess.run(["cvmfs_server", "resign"], check=True)
+
```

Comment:
Nit, but can we return some message like the other endpoints? So that when we run this manually, we have some feedback.